bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/secondgeneration/SecondGenerationHandler.java

```diff
@@ -347,6 +354,12 @@ private final void preSetExecuteConfigurationChanges(HttpClient httpClientHandle
     private final String callURL(String dxsEntriesCall, HttpClient httpClient) {
         String jsonDxsResponse = "";
         try {
+            if (!httpClient.isRunning()) {
+                try {
+                    httpClient.start();
+                } catch (Exception e1) {
+                }
+            }
```

Comment:
Never do that when the binding is using the common HTTP client (common to all bindings).
The common HTTP client will be already started when the binding is started.